### PR TITLE
fix(ci/qcow2): prepare for building qcow2 images automatically

### PR DESCRIPTION
**Pull request by NaddiNadja**
This is necessary for the github actions in this repo:
https://github.com/xnvme/xnvme-qcow2/

### CODE REVIEW

File: cijoe/configs/freebsd-14.toml [hunk 1]
```diff
@@ -49,8 +49,8 @@ system_args.tcp_forward = {host = 4200, guest = 22}
 #system_args.host_share = "{{ local.env.HOME }}/git"
 
 # Used by: qemu.guest_init_using_bootimage.py
-init_using_bootimage.url = "https://refenv.fra1.digitaloceanspaces.com/boot_images/freebsd-14.1-ksrc-amd64.qcow2"
-init_using_bootimage.img = "{{ local.env.HOME }}/images/boot_images/freebsd-14.1-ksrc-amd64.qcow2"
+init_using_bootimage.url = "https://refenv.fra1.digitaloceanspaces.com/boot_images/freebsd-13.1-ksrc-amd64.qcow2"
```
Comment (karlowich) [inline cijoe/configs/freebsd-14.toml]: This should remain `freebsd-14.1` right?

**Review by karlowich — CHANGES_REQUESTED** [review]
It seems like you're only changing something for FreeBSD - so why the changes to the `bootimg-debian` workflow?

**Comment by NaddiNadja** [comment]
> It seems like you're only changing something for FreeBSD - so why the changes to the `bootimg-debian` workflow?

No, this is not about FreeBSD - I don't know why I included the changes in the config, I've removed them now. It's about adding the xnvme provisioning of qemu images from tgz to the workflows. This is necessary for the github actions in this repo:
https://github.com/xnvme/xnvme-qcow2/

**Review by safl — COMMENTED** [review]
It would be great if you added the description from the comments in the commit message: 

```
It's about adding the xnvme provisioning of qemu images from tgz to the workflows. This is necessary for the github actions in this repo.
```
Re-formulated... that is...

**Comment by NaddiNadja** [comment]
> It would be great if you added the description from the comments in the commit message:

done :)

**Review by safl — APPROVED** [review]
LGTM

**Review by karlowich — APPROVED** [review]
lgtm.

(Is it intentional that the freebsd workflow include at least twice the amount of documentation?)

**Comment by NaddiNadja** [comment]
> (Is it intentional that the freebsd workflow include at least twice the amount of documentation?)

@karlowich, it's a copy-paste of the FreeBSD 13 workflow, so I didn't notice the different length of the docstrings. Should I write more in the Debian workflow docstring?

**Comment by karlowich** [comment]
@NaddiNadja I don't think it is super important - but then again, why not.

**Pull request merged**